assets/rsa-pss/README.md

```diff
@@ -0,0 +1,7 @@
+# Generating Test Certificates
```

Comment:
small nit: WDYT about replacing this `.md` documenting the commands to run with a `.sh` that runs them?